packages/sdk-browser/src/sdk-client.ts

```diff
@@ -160,13 +160,28 @@ export class TurnkeyBrowserSDK {
   logoutUser = async (): Promise<boolean> => {
     await removeStorageValue(StorageKeys.CurrentUser);
     await removeStorageValue(StorageKeys.AuthBundle);
+    await removeStorageValue(StorageKeys.SigningSession);
 
     return true;
   };
 
   getAuthBundle = async (): Promise<string | undefined> => {
     return await getStorageValue(StorageKeys.AuthBundle);
   };
+
+  getSigningSession = async (): Promise<SigningSession | undefined> => {
+    const signingSession: SigningSession | undefined = await getStorageValue(StorageKeys.SigningSession);
+    if (signingSession) {
+      if (signingSession.expirationTimestamp > Date.now()) {
+        return signingSession;
+      } else {
+        await removeStorageValue(StorageKeys.SigningSession);
+        return undefined;
+      }
+    } else {
```

Comment:
Minor stylistic thing: we can remove this else branch and just have a final `return undefined` at the end of this function

Reply:
addressed